zk/tests/conftest.py

```diff
@@ -18,20 +19,66 @@
 HERE = os.path.dirname(os.path.abspath(__file__))
 URL = "http://{}:{}".format(HOST, PORT)
 
-
-VALID_CONFIG = {'host': HOST, 'port': PORT, 'expected_mode': "standalone", 'tags': ["mytag"]}
+VALID_CONFIG = {
+    'host': HOST,
+    'port': PORT,
+    'expected_mode': "standalone",
+    'tags': ["mytag"],
+    'timeout': 500,
+}
+
+VALID_SSL_CONFIG = {
+    'host': HOST,
+    'port': PORT,
+    'expected_mode': "standalone",
+    'tags': ["mytag"],
+    'timeout': 500,
+    'enable_ssl': True,
+    'tls_private_key': '/conf/private_key.pem',
+    'tls_ca_cert': '/conf/ca_cert.pem',
```

Comment:
Just realizing that it's gonna be tricky. Most users might be used to use the `keytool` utility directly to update the jks keystore and may not have access to the pem files.

On the server side, the `keystore` is the certificate exposed to clients by the server `trustore` can be empty in most cases if the server doesn't require client authentication.

So users will need to:
- export the certificate of the server from keystore and convert it into a `ca_cert.pem` file. No need to export the private key.
- if using client authentication, generate a new keypair. Add the certificate to the server truststore. Convert the certificate into .pem as well as the certificate's private key and add it to the agent config.

Hope that's clear enough otherwise ping me directly. We should document this for users as the conversion between java certificates and .pem is not the most obvious thing.

Reply:
I was thinking that it would be the user's responsibility to have a correct `cert`/`ca_cert`/`private_key` combination generated from the `keystore` and `truststore` from the server side. I can definitely document the process that I went through to generate `cert`/`ca_cert`/`private_key` from `keystore` and `truststore' in the `README` to make it easier.

Reply:
It's the user responsibility to have a keystore and truststore correctly setup for using zookeeper. But the fact that the agent doesn't use java forces us to use .pem file, which users may have to export manually from their truststore/keystore. Adding instructions for this is important I believe but that can be done in another PR.

Reply:
Added in https://github.com/DataDog/integrations-core/pull/8046.